app/src/main/java/app/mywatchlist/screens/MainScreen.kt

```diff
@@ -0,0 +1,69 @@
+package app.mywatchlist
+
+
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.material.BottomNavigation
+import androidx.compose.material.BottomNavigationItem
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.navigation.NavDestination
+import androidx.navigation.NavDestination.Companion.hierarchy
+import androidx.navigation.NavGraph.Companion.findStartDestination
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.currentBackStackEntryAsState
+import androidx.navigation.compose.rememberNavController
+
+@Composable
+fun MainScreen(){
+    val navController = rememberNavController()
+    Scaffold(
+        bottomBar = { BottomBar(navController = navController) }
+    ) {
+        SetupNavGraph(navController = navController)
+    }
+}
+
+@Composable
+fun BottomBar(navController: NavHostController){
```

Comment:
I'd perhaps move `BottomBar` and `RowScope.AddItem` to a separate file and make the `AddItem` private.